README.md

```diff
@@ -208,6 +208,17 @@ To delete a nodegroup, run:
 
 ```
 eksctl delete nodegroup --cluster=<clusterName> --name=<nodegroupName>
+```
+### Enable Autoscaling
+
+```bash
+
+eksctl create cluster --asg-access=true
```

Comment:
Nit but I think `=true` is redundant here :)

Reply:
I'll fix this on master, thanks!